org.musicbrainz.Picard.appdata.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.musicbrainz.Picard</id>
+  <project_license>GPL-2.0+</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Picard</name>
+  <summary>MusicBrainz's music tagger</summary>
+
+  <description>
+    <p>
+      Do you need to clean up your music library? Picard is an open-source
+      cross-platform music tagger by MusicBrainz. It has the ability to
+      identify audio files without any metadata.
+    </p>
+    <p>
+      A variety of plugins are available and you can even write your own.
+      Picard supports a wide range of audio formats and can also lookup an
+      entire CD for you.
+    </p>
+    <p>
+      Tagging audio files has never been easier.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">org.musicbrainz.Picard.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">https://www.daitauha.fr/static/screenshots/picard/main.png</screenshot>
+  </screenshots>
+
+  <url type="homepage">https://picard.musicbrainz.org/</url>
+  <url type="bugtracker">https://tickets.metabrainz.org/projects/PICARD/issues/</url>
+  <url type="faq">https://picard.musicbrainz.org/docs/faq/</url>
+  <url type="help">https://community.metabrainz.org/c/picard</help>
```

Comment:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url suggests that the "help" type should be for "an online user's reference, a software manual or help page." I.e., documentation of some kind, not a support forum. Is it de facto used to link to forums (ie., did you see other AppStream files doing this) and the spec just doesn't reflect this (yet)?

Reply:
I agree with you. Its a mistake I made. I am correcting it.
```suggestion
  <url type="help">https://picard.musicbrainz.org/docs/</url>
```